plugins/lang-painless/src/main/java/org/elasticsearch/painless/PainlessLexer.java

```diff
@@ -1,19 +1,16 @@
 // ANTLR GENERATED CODE: DO NOT EDIT
 package org.elasticsearch.painless;
 
-import org.antlr.v4.runtime.CharStream;
+    import java.util.Set;
```

Comment:
It's a personal preference of mine to have the Lexer file look like this --

@header {
    import java.util.Set;
}

Instead it can be changed to look like this --

@header {
import java.util.Set;
}

However, I felt that it was more important to make the created content readable over the auto-generated content.